server/controllers/rentals/deleteRental.js

```diff
@@ -0,0 +1,19 @@
+const { deleteRentalQuery } = require('../../database/queries');
+const boomify = require('../../utils/boomify');
+
+const deleteRental = async (req, res, next) => {
+  const { rentalId } = req.params;
+  try {
+    const { rows } = await deleteRentalQuery(rentalId);
+    if (rows.length) {
+      res.json({ status: 200, message: 'rental deleted successfully !' });
```

Comment:
```suggestion
      return res.json({ status: 200, message: 'rental deleted successfully !' });
```
we should return to stop the function, otherwise it will continue and throw an error